### ISSUE TITLE
Test VS Live Server/Preview - opening a file using a link from the terminal

### ISSUE THREAD

**Issue by lszomoru** [microsoft/vscode#127459]
Testing #127164

When clicking on the link to a file in the terminal, I am wondering whether we should be opening this file in a separate editor group than the preview itself. This way, you can see both the file that you have chosen to open as well as the preview itself. We could also make this configurable to give users more flexibility on how the files are being opened.

**Comment by rzhao271** [comment]
@andreamah what are some steps to verify this feature?

**Comment by andreamah** [comment]
I just pushed some fixes to the terminal link provider to the newest version a few minutes ago, so please upgrade to v0.1.7!

Steps:
1. Run the task `Live Preview: Run Server --verbose`. 
2. Open the internal preview of an html page and interact with some of the links to navigate to other pages and load content. The task terminal should be logging server traffic.
3. Ensure that the active editor is the preview webview.
4. Try clicking on the filenames in the terminal as you would with a hyperlink (ctrl+click)
5. The file should either open up in an existing editor group beside the webview or open a new editor group to the right in a single-column scenario. It should not open in the same editor group as the embedded preview.

**Comment by rzhao271** [comment]
I got the tab to show up in the same editor group in the following case:

1. Have two editor groups, one with a file, and one with the preview.
2. Close the file. This closes that editor group, and the preview remains.
3. Click a link in the terminal.

![Screencap demonstrating steps above](https://user-images.githubusercontent.com/7199958/124305620-33d28780-db1a-11eb-9037-3b65da12365e.gif)

**Comment by andreamah** [comment]
Good catch! I checked and seems that the panel's viewcolumn was not updated by the time that the open was performed in this specific case... I'll need to look into this.
